### DISCUSSION TITLE
New labels for issues and pull requests

### DISCUSSION THREAD

**Discussion by ghost1372** [HandyOrg/HandyControl#568] [General]
Thanks to GitHub and you❤
@NaBian  I suggest adding a few new labels to the issues and pull request section to make it easier to manage and handle issues and prs

- Roadmap
- Documentation   
- Localization
- In-PR     (This issue has a related PR)
- In progress     (This issue or work-item is under development)
- New Feature
- Need More Info
- DO-NOT-MERGE    (Pull Request should not be merged)

**Comment by NaBian** [comment]
pretty good
